src/main/k8s/production.yml

```
@@ -28,7 +28,9 @@ spec:
           requests:
             cpu: 200m
             memory: 512Mi
-        args: ["-Dconfig.secret.path=/run/secrets/git-eca-rest-api/secret.properties"]
+        env: 
+           - name: CONFIG_SECRET_PATH
+             value: "/run/secrets/git-eca-rest-api/secret.properties"
```

This is too much indented (yeah, I hate yaml as well), it should be:

```suggestion
        env: 
        - name: CONFIG_SECRET_PATH
          value: "/run/secrets/git-eca-rest-api/secret.properties"
```